src/app/components/challenge/challenge.component.html

```diff
@@ -1,150 +1,153 @@
-<div class="mid-container">
-    <div class="challenge-container">
-        <div class="challenge-card">
-            <div class="top-card-container ev-card-panel">
-                <div class="row card-img-row">
-                    <div class="col-lg-2 col-md-3 col-sm-12">
-                        <div class="image-container">
-                            <img [src]="challenge['image']" width="100%">
-                        </div>
-                    </div>
-                    <div class="col-lg-7 col-md-6">
-                        <div class="card-content">
-                            <p>
-                                <span style="line-height: 1.5" class="fs-18">
-                                    <strong style="color: rgba(0, 0, 0, 0.87);"
-                                        class="fw-semibold">
-                                        {{challenge['title']}}
-                                    </strong>
-                                </span>
-                                &nbsp;
-                                <span *ngIf="isChallengeHost">
-                                    <a style="padding-right: 4px;"
-                                        class="pointer"
-                                        (click)="editChallengeTitle()">
-                                        <i class="fa fa-pencil"
-                                            aria-hidden="true"></i>
-                                    </a>
-                                    <a class="pointer"
-                                        (click)="deleteChallenge()">
-                                        <i class="fa fa-trash red-text"
-                                            aria-hidden="true"></i>
-                                    </a>
-                                </span>
-                                <br>
-                                <span style="line-height: 1.5"
-                                    *ngIf="challenge.creator.team_url != ''"><strong
-                                        class="text-light-black fw-semibold
-                                        fs-15">Organized by:</strong><a
-                                        class="orange-text"
-                                        href="{{challenge['creator']['team_url']}}"
-                                        target="_blank">
-                                        {{challenge['creator']['team_name']}}
-                                    </a>
-                                </span>
-                                <span style="line-height: 1.5"
-                                    *ngIf="challenge.creator.team_url == ''"><strong
-                                        class="text-light-black fw-semibold
-                                        fs-15">Organized by:</strong>
-                                    {{challenge['creator']['team_name']}}
-                                </span>
+<div class="web-container" [style.center]="authService.isAuth">
```

Comment:
You need to center the container when user is not logged in. Currently the UI is right aligned on the challenge overview page when the user is not loggeed in. @nsjcorps Please look into it.

Reply:
Done @Sanji515

Reply:
PTAL @Sanji515

Reply:
Still the same @nsjcorps